js-intro.js

```diff
@@ -8,32 +9,54 @@ var mantra = "Be the dev"
 // 1a. Write the code that determines if there is a 'B' in mantra.
 
 
+console.log(mantra.includes("B"))
 
-// 1b. Write the code that determines if there is an 'x' in mantra.
 
+// 1b. Write the code that determines if there is an 'x' in mantra.
 
+console.log(mantra.includes("x"))
 
 // 1c. Write the code that determines if there is a 'v' in mantra.
 
+console.log(mantra.includes("v"))
 
 
 // 1d. STRETCH: Write a function that can take two arguments - a string and a letter and returns a sentence telling the user if the letter exists within the string.
-
-
-
+const exists = (x, y) =>{
+    let str = "sentence"
+    let letter = "e"
+    if (str.includes("e")){
+        return true
+    }else{
+        return false
+    }
+}
+
+console.log(exists())
 // ------------------------------ Consider the following variables:
```

Comment:
You have the right idea here, but the string and the letter should come from the arguments. 
const exists = (str, letter) =>{
    if (str.includes(letter)){
        return true
    }else{
        return false
    }
}
console.log(exists("sentence", "e"))